pyscriptjs/tests/test_01_basic.py

```diff
@@ -21,7 +22,8 @@ def test_pyscript_hello(self):
         """
         self.write("hello.html", doc)
         self.goto("hello.html")
+        # I wonder if we need to call wait_for_load in every single test?
+        # It's only relevant to call it on tests that are interfering somehow with the way we load the page, IMO
         wait_for_load(self.page)
         content = self.page.content()
-        # XXX write a test with the DOM
```

Comment:
you should have not removed the `XXX` because it has not been resolved yet.
Currently the test is broken: for example, imagine that we forgot to add the relevant `<script>` tag:
```diff
diff --git a/pyscriptjs/tests/test_01_basic.py b/pyscriptjs/tests/test_01_basic.py
index 861dfc3..ad1f0e3 100644
--- a/pyscriptjs/tests/test_01_basic.py
+++ b/pyscriptjs/tests/test_01_basic.py
@@ -11,8 +11,6 @@ class TestBasic(PyScriptTest):
         doc = f"""
         <html>
           <head>
-              <link rel="stylesheet" href="{self.http_server}/build/pyscript.css" />
-              <script defer src="{self.http_server}/build/pyscript.js"></script>
           </head>
           <body>
             <py-script>
@@ -25,6 +23,6 @@ class TestBasic(PyScriptTest):
         self.goto("hello.html")
         # I wonder if we need to call wait_for_load in every single test?
         # It's only relevant to call it on tests that are interfering somehow with the way we load the page, IMO
-        wait_for_load(self.page)
+        #wait_for_load(self.page)
         content = self.page.content()
         assert "hello pyscript" in content
```
If you apply the diff above, you notice that the test passes even if pyscript does not run at all, which means that the test is broken (because it is supposed to test that pyscript executes the `print()` inside the tag).
Understanding why the test passes anyway is left as an exercise to the reader :)